content/00.build/30.zksync-sso/25.interfaces.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Interfaces
+description: Get familiar with ZKsync SSO interfaces
+---
+
+While the internal functionality can be quite complex, the external interfaces for developers are very simple.
+This provides some technical information on how these components can be used.
+
+### Auth Server
+
+The entrypoint to the SDK is via the ``zksyncAccountConnector`` to connect to the *Auth Server*.
+This takes the auth domain, session, and basic app information (name, icon). The fully expanded type looks like:
+
+```ts
+type ZKsyncAccountConnectorOptions = {
+  metadata?: {
+    name: string,
+    icon?: string,
+  },
+  session?: {
```

Comment:
A bunch of fields here were renamed in @JackHamer09's PR

Reply:
updated!
* contractCalls
* address
* transfers